### ISSUE TITLE
The notification panel

### ISSUE THREAD

**Issue by DaxCorner** [jeancsanchez/JCPlayer#71]
Good day.
1) Play  button not work.
2) Constantly notification sounds like in messages

https://www.youtube.com/watch?v=m1mUjsXxkuU&frags=pl%2Cwn

Android 8.1

Thanks

**Comment by jeancsanchez** [comment]
I have not tested on Android 7 or higher.

**Comment by rohankandwal** [comment]
you need to update the 'createNotificationPlayer' function in JcNotificationPlayer class. Use the following code :-

```
 fun createNotificationPlayer(title: String?, iconResourceResource: Int) {
    this.title = title
    this.iconResource = iconResourceResource
    val openUi = Intent(context, context.javaClass)
    openUi.flags = Intent.FLAG_ACTIVITY_SINGLE_TOP

    val priority = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
      NotificationCompat.PRIORITY_HIGH
    } else {
      NotificationCompat.PRIORITY_DEFAULT
    }
    val notificationIntent = Intent(context, context::class.java)
    notificationIntent.action = Intent.ACTION_MAIN
    notificationIntent.addCategory(Intent.CATEGORY_LAUNCHER)
    notificationIntent.addFlags(
        Intent.FLAG_ACTIVITY_SINGLE_TOP or Intent.FLAG_ACTIVITY_REORDER_TO_FRONT or Intent.FLAG_ACTIVITY_CLEAR_TOP)

    notification = NotificationCompat.Builder(context, NOTIFICATION_CHANNEL)
        .setSmallIcon(iconResourceResource)
        .setCategory(CATEGORY_MUSIC)
        .setSound(null)
        .setLargeIcon(BitmapFactory.decodeResource(context.resources, iconResourceResource))
        .setPriority(NotificationCompat.PRIORITY_DEFAULT)
        .setContent(createNotificationPlayerView())
        .setContentIntent(PendingIntent.getActivity(context, NOTIFICATION_ID, notificationIntent,
            PendingIntent.FLAG_CANCEL_CURRENT))
        .setAutoCancel(false)
        .setOngoing(true)
        .setPriority(NotificationCompat.PRIORITY_HIGH)
        .setDefaults(Notification.DEFAULT_LIGHTS)
        .setVibrate(LongArray(1) { 0L })
        .setPriority(priority)
        .build()

    @RequiresApi(Build.VERSION_CODES.O)
    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
      val channel = NotificationChannel(NOTIFICATION_CHANNEL, NOTIFICATION_CHANNEL,
          NotificationManager.IMPORTANCE_DEFAULT)
      channel.enableVibration(false)
      channel.description = CATEGORY_MUSIC
      channel.lockscreenVisibility = VISIBILITY_PUBLIC
      channel.setSound(null, null)
      notificationManager.createNotificationChannel(channel)
    }

    try {
      notification?.let { notificationManager.notify(NOTIFICATION_ID, it) }
    } catch (e: Exception) {
      e.printStackTrace()
    }
  }
```

This will remove any sounds that are coming and work well in all Android versions. On clicking on notification, the activity from which you started JcPlayer, will open.

**Comment by jeancsanchez** [comment]
@rohankandwal Thanks. I'll test it.

**Comment by jeancsanchez** [comment]
@lab76pro Fixed on version  `2.6.14`
